requirements.txt

```diff
@@ -0,0 +1,3 @@
+# python
```

Comment:
what's this comment for?

Reply:
did this subconsciously, kind of doesn't make sense now. I'll remove it.